target/linux/ipq40xx/base-files/etc/board.d/01_leds

```diff
@@ -41,6 +41,9 @@ engenius,ens620ext)
 	ucidef_set_led_netdev "lan1" "LAN1" "${boardname}:green:lan1" "eth0"
 	ucidef_set_led_netdev "lan2" "LAN2" "${boardname}:green:lan2" "eth1"
 	;;
+linksys,ea8300)
+	ucidef_set_led_usbport "usb" "USB" "ea8300:green:usb" "usb1-port1"
```

Comment:
I think you could convert this ucidef_ to device-tree method. See 
ef5ff08662532a6b1770214569e4c27b05e1dc35